conda-recipe/meta.yaml

```diff
@@ -24,12 +24,17 @@ requirements:
         - make  # [unix]
         - ninja  # [win]
         - numpy >=1.17
+        - pytest
```

Comment:
Like @PokhodenkoSA says, remove these two lines. Having packages listed as test-time dependencies is sufficient.